libr/bin/bin.c

```diff
@@ -527,6 +527,7 @@ static void r_bin_object_free(void /*RBinObject*/ *o_) {
 	}
 	r_bin_info_free (o->info);
 	r_bin_object_delete_items (o);
```

Comment:
What do you mean? That code is in the `r_bin_object_free` function, it would become a recursive call.